django/test/client.py

```diff
@@ -594,12 +591,11 @@ def login(self, **credentials):
         Sets the Factory to appear as if it has successfully logged into a site.
 
         Returns True if login is possible; False if the provided credentials
-        are incorrect, or the user is inactive, or if the sessions framework is
-        not available.
+        are incorrect, or the user is inactive.
```

Comment:
"or the user is inactive" can be removed too (should have been removed in 107165c4b04f4e5a830a60b6c66b2e5d8fb1d242)